style/style.css

```
@@ -106,6 +106,7 @@ pre {
 img {
     display: block;
     margin: auto;
+	max-width: 100%;
```

Please use 4 spaces to indent, not tabs :)

```suggestion
    max-width: 100%;
```